app/theme/client/imports/components/message-box.css

```diff
@@ -144,6 +144,12 @@
 		& .rc-input__icon-svg--plus {
 			transition: transform 0.1s linear;
 		}
+		&:hover,
+		&:focus,
+		&.active {
+			color: red;
```

Comment:
Please don't use a static red color, this doesn't go well with the design. Also we have dynamic colors, I mean, user can change colors of UI elements, so we should not provide static color values, instead do it like [this](https://github.com/RocketChat/Rocket.Chat/blob/978cd3a845b01a164e5b9fd7b82ad41d9a3ef4c8/app/theme/client/imports/components/message-box.css#L39). The variable `message-box-markdown-hover-color` contains the color selected by user.